.github/workflows/lint.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.21.x
+          go-version: 1.23.x
```

Comment:
These changes are needed due to updating to the latest version of k6, which works with dependencies that require at least v1.22 of go.

Reply:
I'm fine with this, but should this part go into a precursory PR?

Reply:
Yeah, i realised that after doing it. Next time!